servers/rendering/rendering_device.cpp

```diff
@@ -4549,6 +4629,22 @@ void RenderingDevice::draw_list_draw_indirect(DrawListID p_list, bool p_use_indi
 	_check_transfer_worker_buffer(buffer);
 }
 
+void RenderingDevice::draw_list_set_viewport(DrawListID p_list, const Rect2 &p_rect) {
```

Comment:
This is nice to have, but it seems like it is unused (I think it was originally part of the render pass optimizations right?). Let's leave it in at any rate as I want to use it for an optimization I have in mind for PointLight2Ds

Reply:
I looked into this, and you're right. This function of code was added by the Render Pass Optimization which was not included in this PR.

Anyway, soon another PR will be incoming w/ it.